membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -164,12 +164,12 @@ object AccountDetails {
             "end" -> endDate,
             "nextPaymentPrice" -> paymentDetails.nextPaymentPrice,
             "nextPaymentDate" -> paymentDetails.nextPaymentDate,
+            "potentialCancellationDate" -> paymentDetails.nextInvoiceDate,
```

Comment:
Why did you choose `potentialCancellationDate` over `nextInvoiceDate`?

Reply:
I wanted it to be clear why it was useful for the user rather than where it came from, (the reason this field is added today is because we reused the nextPaymentDate on the client side as the cancellation date).

As a later PR, we should probably flip this to use the actual cancellation calculation (which is in the "decideCancellationDate" function) as that is what's actually used when you ask MDAPI to cancel a sub.

Happy to take suggestions though.